docs/user/exercises/modeling.rst

```diff
@@ -76,13 +76,22 @@ The following screenshot illustrates the first section of the form. It consists
 The following screenshot illustrates the second section of the form. It consists of:
 
 - **Enable automatic assessment suggestions**: When enabled, Artemis tries to automatically suggest assessments for diagram elements based on previously graded submissions for this exercise.
-- **Enable feedback suggestions from Athena**: When enabled, Artemis tries to automatically suggest assessments for diagram elements using the Athena service.
 - **Problem Statement**: The task description of the exercise as seen by students.
 - **Assessment Instructions**: Instructions for instructors while assessing the submission.
 
     .. figure:: modeling/create-modeling-exercise-form-2.png
               :align: center
 
```

Comment:
I think we can also update the figure here - and link your below descriptions to the related checkboxes 
<img width="1471" alt="Screenshot 2025-04-04 at 14 40 05" src="https://github.com/user-attachments/assets/56a44857-f1f6-4796-810e-db5551292b44" />